test/jit/test_list_dict.py

```diff
@@ -1645,9 +1645,9 @@ def test_dictliteral_is_typed_from_annotation():
 
         def test_dictcomprehension_is_typed_from_annotation():
             metasyntactics = ["foo", "bar", "baz"]
-            x: Dict[str, Optional[int]] = {
+            x: Dict[str, Optional[int]] = {  # noqa: C420, RUF025
                 word: None for word in metasyntactics
-            }  # noqa: RUF025
+            }
```

Comment:
why not
```python
  x: Dict[str, Optional[int]] = dict.fronkeys(metasyntactics)
```

Reply:
@lezcano Because it's testing JIT features and I didn't want to touch it.